requirements/constraints.txt

```diff
@@ -1,8 +1,11 @@
-# Stay on the same Django major version as our other services
-Django<3
+# Version constraints for pip-installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why.  Ideally,
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
 
-# stevedore 2.0.0 doesn't install on python 3.5 (circa 2020-06-10)
```

Comment:
These constraints are not needed any more because the repo has been upgraded to use `Python3.8`.